### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by larspalo**
Changes to reflect the new state of GrandOrgue development.

### CODE REVIEW

**Review by oleg68 — APPROVED** [review]

File: src/grandorgue/GrandOrgueFrame.cpp [hunk 1]
```diff
@@ -205,6 +204,8 @@ GOrgueFrame::GOrgueFrame(
   m_file_menu->Append(ID_FILE_IMPORT_COMBINATIONS, _("Import &Combinations"), wxEmptyString, wxITEM_NORMAL);
   m_file_menu->Append(ID_FILE_EXPORT, _("&Export Settings/Combinations"), wxEmptyString, wxITEM_NORMAL);
   m_file_menu->AppendSeparator();
+  m_file_menu->Append(ID_SETTINGS, wxT("&Settings..."), wxEmptyString, wxITEM_NORMAL);
```
Comment (rousseldenis) [inline src/grandorgue/GrandOrgueFrame.cpp:207]: @larspalo Usually, it's better not to mix concepts in a same PR. So, if you want to change things that are not linked together, it's better to do separate PR's.

**Comment by larspalo** [comment]
Yes, I didn't intend for them to be mixed. But somehow it did anyway...

**Comment by larspalo** [comment]
Anyway, this is not working that well - it's just taking too long. I'll remove this PR and then we can just commit Olegs and I'll change the necessary things.
